content/sensu-go/5.16/api/mutators.md

```diff
@@ -107,7 +135,8 @@ The `/mutators/:mutator` API endpoint provides HTTP GET access to [mutator data]
 In the following example, querying the `/mutators/:mutator` API endpoint returns a JSON map that contains the requested [`:mutator` definition][1] (in this example, for the `:mutator` named `example-mutator`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators/example-mutator -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/mutators/example-mutator
```

Comment:
requires '\' at the end of the line for multi line command. 'curl -X GET' should probably be on its own line like the previous example.